ccd-definition/CaseField.json

```diff
@@ -163,5 +163,38 @@
     "HintText": "This is optional - the claimant will see your answer.",
     "FieldType": "TextArea",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionAccepted",
+    "Label": "Do you agree to extend the deadline to this date?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionCounter",
+    "Label": "Would you like to set a different extended deadline?",
+    "FieldType": "YesOrNo",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionCounterDate",
+    "Label": "New deadline",
+    "HintText": "This can't be later than 28 working days after the current deadline. All deadlines require a response before 4pm on the deadline day.",
+    "FieldType": "Date",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "extensionResponse",
```

Comment:
@swalker125 - specifically on this attribute, what's in this part of the response that warrants an additional attribute outside of the others being configured?

Reply:
to match the design, please see the JIRA ticket and prototype.